README.md

```diff
@@ -94,6 +94,8 @@ Download and install the latest [nRF5x-Command-Line-Tools](https://www.nordicsem
 
 ### Linux and macOS
 
+> J-Link driver needs to be separately installed on Linux and macOS, download and install appropriate package for your operating system from [SEGGER](https://www.segger.com/downloads/jlink) under the section *J-Link Software and Documentation Pack*.
```

Comment:
In my opinion it makes it look like the text is quoted from somewhere else. But I'll let you make the call.